assignment.js

```diff
@@ -66,6 +72,23 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+//Creating array and largest number variables
+const numbers = [3, 48, 29, 12, 18]
+let currentLargestNumber = 0
+
+
+//Using for each loop and applying to numbers array
+numbers.forEach(function(number) {
+
+  //Us an if statement to check the currentLargestNumber variable
+  if(currentLargestNumber < number ) {
+    //If the variable is not the largest, it reassigns the variable to the new larger number
+    currentLargestNumber = number
+  }
+})
+
+console.log(currentLargestNumber)
+
```

Comment:
@cswebdesign Nice work here using the forEach method to loop through each element and using the conditionals to compare the values and check for the largest value.
Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L73)